vendor/views/vendor_admin.py

```diff
@@ -387,3 +388,48 @@ def get_context_data(self, *args, **kwargs):
         context = super().get_context_data(**kwargs)
         context['title'] = _("Payments with no Order Items")
         return context
+
+
+class PaymentProcessorSiteConfigsListView(ListView):
+    template_name = 'vendor/manage/processor_site_config_list.html'
+    model = SiteConfigModel
+    queryset = SiteConfigModel.objects.all()
+
+
+class PaymentProcessorFormView(FormView):
+    template_name = 'vendor/manage/processor_site_config.html'
+    form_class = PaymentProcessorForm
+
+    def get_success_url(self):
+        return reverse('vendor_admin:vendor-processor')
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        processor_config = PaymentProcessorSiteConfig()
```

Comment:
pass in site via get_site_from_request?

Reply:
this are just helper functions, if the site is not set it will use the Site.objects.get_current()